tests/IntegrationTests/WorkflowExecutor.IntegrationTests/Support/Assertions.cs

```diff
@@ -99,13 +100,13 @@ public void AssertWorkflowIstanceMatchesExpectedTaskStatusUpdate(WorkflowInstanc
             updatedWorkflowInstance.Tasks[0].Status.Should().Be(taskExecutionStatus);
         }
 
-        public void AssertPagination<T>(int count, string queries, T Response)
+        public static void AssertPagination<T>(int count, string queries, T? Response)
         {
-            var data = Response.GetType().GetProperty("Data").GetValue(Response, null) as ICollection;
-            var totalPages = Response.GetType().GetProperty("TotalPages").GetValue(Response, null);
-            var pageSize = Response.GetType().GetProperty("PageSize").GetValue(Response, null);
-            var totalRecords = Response.GetType().GetProperty("TotalRecords").GetValue(Response, null);
-            var pageNumber = Response.GetType().GetProperty("PageNumber").GetValue(Response, null);
+            var data = Response?.GetType()?.GetProperty("Data")?.GetValue(Response, null) as ICollection;
```

Comment:
assigning `Response?.GetType()?` to a variable could simplify this code a bit.

Reply:
Changed